source/libs/utils.ts

```diff
@@ -138,3 +138,5 @@ export function reportBug(featureName: string, bugName: string): void {
 	console.log('Find existing issues:\n' + String(issuesUrl));
 	console.log('Open new issue:\n' + String(newIssueUrl));
 }
+
+export const noOp = (): void => {}; // eslint-disable-line @typescript-eslint/no-empty-function
```

Comment:
```suggestion
export const noop = (): void => {}; // eslint-disable-line @typescript-eslint/no-empty-function
```

It's more common to write it like this.

https://api.jquery.com/jQuery.noop/

Reply:
This is the smallest non-optimization ever. Can we just leave it inlined?